e2e/config.js

```diff
@@ -6,18 +6,18 @@ module.exports = {
     url: 'http://localhost:5555'
   },
   url: {
-    // manageCase: process.env.URL || 'https://manage-case-int.demo.platform.hmcts.net/',
-    // authProviderApi: process.env.SERVICE_AUTH_PROVIDER_API_BASE_URL || 'http://rpe-service-auth-provider-demo.service.core-compute-demo.internal',
-    // ccdDataStore: process.env.CCD_DATA_STORE_URL || 'http://ccd-data-store-api-demo.service.core-compute-demo.internal',
-    // dmStore:process.env.DM_STORE_URL || 'http://dm-store-demo.service.core-compute-demo.internal',
-    // idamApi: process.env.IDAM_API_URL || 'https://idam-api.demo.platform.hmcts.net',
-    // civilService: process.env.CIVIL_SERVICE_URL || 'http://civil-service-demo.service.core-compute-demo.internal'
-    manageCase: process.env.URL || 'http://localhost:3333',
-    authProviderApi: process.env.SERVICE_AUTH_PROVIDER_API_BASE_URL || 'http://localhost:4502',
-    ccdDataStore: process.env.CCD_DATA_STORE_URL || 'http://localhost:4452',
-    dmStore: process.env.DM_STORE_URL || 'http://dm-store:8080',
-    idamApi: process.env.IDAM_API_URL || 'http://localhost:5000',
-    civilService: process.env.CIVIL_SERVICE_URL || 'http://localhost:4000',
+    //manageCase: 'https://xui-civil-ccd-pr-1287.service.core-compute-preview.internal',
+    //authProviderApi: 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal',
+    //ccdDataStore: 'https://ccd-data-store-api-civil-ccd-pr-1287.service.core-compute-preview.internal',
+    //dmStore: 'http://dm-store-aat.service.core-compute-aat.internal',
+    //idamApi: 'https://idam-api.aat.platform.hmcts.net',
+    //civilService: 'https://civil-ccd-pr-1287.service.core-compute-preview.internal'
+    manageCase: 'https://xui-civil-ccd-pr-1327.service.core-compute-preview.internal',
+    authProviderApi: 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal',
+    ccdDataStore: 'https://ccd-data-store-api-civil-ccd-pr-1327.service.core-compute-preview.internal',
+    dmStore: 'http://dm-store-aat.service.core-compute-aat.internal',
+    idamApi: 'https://idam-api.aat.platform.hmcts.net',
+    civilService: 'https://civil-ccd-pr-1327.service.core-compute-preview.internal'
```

Comment:
if we replace everything with absolute urls it will affect us when we try to run the tests locally

Reply:
I will leave TODOs here, I have used absolute URLs based on Mounika's suggestions, but they should be temporary.
In fact, I'll see if the PR runs without them.

Reply:
Changed them to the original that allows for local tests, as the change appeared to be unnecessary.